modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/migrate/MigrateDeployProcess.java

```diff
@@ -0,0 +1,37 @@
+package org.eclipse.xpanse.modules.workflow.migrate;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.activiti.engine.RuntimeService;
+import org.activiti.engine.delegate.DelegateExecution;
+import org.activiti.engine.delegate.JavaDelegate;
+import org.eclipse.xpanse.modules.deployment.DeployService;
+import org.eclipse.xpanse.modules.models.service.deploy.CreateRequest;
+import org.eclipse.xpanse.modules.workflow.utils.SpringUtil;
+import org.springframework.stereotype.Component;
+
+/**
+ * Migration process deployment service processing class.
+ */
+@Slf4j
+@Component
+public class MigrateDeployProcess implements Serializable, JavaDelegate {
+
+    /**
+     * Methods when performing deployment tasks.
+     */
+    @Override
+    public void execute(DelegateExecution execution) {
+        String processInstanceId = execution.getProcessInstanceId();
+        RuntimeService runtimeService = SpringUtil.getObject(RuntimeService.class);
```

Comment:
do we really need to use `SpringUtil` here to get the `RuntimeService` and `DeployService` beans?
Since `MigrateDeployProcess`  is also a spring bean, can we not simply autowire them.

Reply:
@swaroopar  Hi swaroopar , I'm very sorry. I also automatically assembled RuntimeService and DeployService in MigrateDeployProcess at the beginning, but the instance objects of RuntimeService and DeployService obtained were null.

Reply:
@WangLiNaruto then there is some other problem. we must check why it is returned null.